Gopkg.toml

```diff
@@ -24,12 +24,13 @@
 
 [[constraint]]
   name = "github.com/ethereum/go-ethereum"
-  version = "=v1.8.16"
+  version = "=v1.8.17"
   source = "github.com/status-im/go-ethereum"
 
 [[constraint]]
   name = "github.com/status-im/whisper"
-  version = "=v1.2.0"
+  revision = "64c74e62c161909003e74edeae8563e803f99329"
+#  version = ">=v1.2.0"
```

Comment:
it seems that @adambabik already created tag with commit that is compatible with geth 1.8.17 - https://github.com/status-im/whisper/tree/v1.3.0